### PR TITLE
Fixes #3211: Old migration was repaired.

### LINKED ISSUE #3211: October:down doesn't work, because migration down is incorrect.
`php artisan october:down` 

This error occurred on sqlite database, but it seems it will be reproduced for other databases too.

##### Expected behavior
Migration will be downgraded correctly without errors.

##### Actual behavior
```
[Illuminate\Database\QueryException]
  SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL (SQL: alter table "jobs"
   add column "reserved" integer not null)
```

```
[Doctrine\DBAL\Driver\PDOException]
  SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL
```

```
[PDOException]
  SQLSTATE[HY000]: General error: 1 Cannot add a NOT NULL column with default value NULL
```
##### Reproduce steps
On fresh installation

1.  `php artisan october:up`
2. `php artisan october:down`

##### October build
```
composer show october/system
name     : october/system
descrip. : System module for October CMS
keywords : october, october cms, system
versions : * v1.0.426
type     : october-module
license  : MIT License (MIT) (OSI approved) https://spdx.org/licenses/MIT.html#licenseText
source   : [git] https://github.com/octoberrain/system.git 281a48a55805804f7be03c06b8ddb94df0bb8ceb
dist     : [zip] https://api.github.com/repos/octoberrain/system/zipball/281a48a55805804f7be03c06b8ddb94df0bb8ceb 281a48a55805804f7be03c06b8ddb94df0bb8ceb
names    : october/system
```

### PR DESCRIPTION
**Pull request by NemoD503**
It's impossible now to downgrade whole project, because when you add new field to existiing table with existing rows, it must be nullable or have default value. Just for this case `nullable` it's enough.

### CODE REVIEW

**Comment by NemoD503** [comment]
@munxar what You think about this change for old migration, is it normal?

File: modules/system/database/migrations/2017_10_01_000022_Db_Jobs_FailedJobs_Update.php [hunk 1]
```diff
@@ -30,7 +30,7 @@ public function up()
     public function down()
     {
         Schema::table($this->getTableName(), function (Blueprint $table) {
-            $table->tinyInteger('reserved')->unsigned();
+            $table->tinyInteger('reserved')->unsigned()->nullable();
```
Comment (LukeTowers) [inline modules/system/database/migrations/2017_10_01_000022_Db_Jobs_FailedJobs_Update.php]: This should not be nullable, instead it should be `->default(0);`

**Comment by LukeTowers** [comment]
This pull request has been made to the wrong branch. Please review [the contributing guidelines](https://github.com/octobercms/october/blob/master/CONTRIBUTING.md#pull-requests) as all PRs need to be made to the develop branch.

I'll fix it for you this time, but please ensure you make any future PRs to the develop branch, not the master branch.

**Comment by NemoD503** [comment]
Sorry for wrong branch,  it was unintentional..
I've fixed to default value.

**Pull request merged**